net-mgmt/telegraf/src/opnsense/service/templates/OPNsense/Telegraf/telegraf.conf

```diff
@@ -173,6 +173,7 @@
 
 {% if helpers.exists('OPNsense.telegraf.input.pf') and OPNsense.telegraf.input.pf == '1' %}
 [[inputs.pf]]
+use_sudo = true
```

Comment:
wouldn't this require to set up sudo as "no password", or how will this work?

Reply:
> Change the ownership and permissions for /dev/pf such that the user telegraf runs at can read the /dev/pf device file. This is probably not that good of an idea either.

This seems like the better option, we already do it for web proxy

Reply:
Oh, I see. Yet I wouldn't want to rely on sudo for this.

Can you try this instead?

    # /usr/sbin/pw groupmod proxy -m telegraf

Reply:
It's working :+1: